arm-recoveryservices/2016-06-01/swagger/registeredidentities.json

```diff
@@ -0,0 +1,394 @@
+          "description": "Certificate friendlyname.",
+          "type": "string"
+        },
+        "issuer": {
+          "description": "Certificate issuer.",
+          "type": "string"
+        }
+      },
+      "x-ms-discriminator-value": "AzureActiveDirectory"
+    },
+    "ResourceCertificateAndACSDetails": {
+      "description": "Certificate details representing the Vault credentials for ACS.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ResourceCertificateDetails"
+        }
+      ],
+      "properties": {
+        "certificate": {
+          "format": "byte",
+          "description": "The base64 encoded certificate raw data string.",
+          "type": "string"
+        },
+        "resourceId": {
+          "format": "int64",
+          "description": "Vault resource id.",
+          "type": "integer"
+        },
+        "globalAcsNamespace": {
+          "description": "ACS namespace name - tenant for our service.",
+          "type": "string"
+        },
+        "globalAcsHostName": {
+          "description": "Acs mgmt host name to connect to.",
+          "type": "string"
+        },
+        "globalAcsRPRealm": {
+          "description": "Global ACS namespace RP realm.",
+          "type": "string"
+        },
+        "subject": {
+          "description": "Certificate Subject Name.",
+          "type": "string"
+        },
+        "validFrom": {
+          "format": "date-time",
+          "description": "Certificate Validity start Date time.",
+          "type": "string"
+        },
+        "validTo": {
+          "format": "date-time",
+          "description": "Certificate Validity End Date time.",
+          "type": "string"
+        },
+        "thumbprint": {
+          "description": "Certificate thumbprint.",
+          "type": "string"
+        },
+        "friendlyName": {
+          "description": "Certificate friendlyname.",
+          "type": "string"
+        },
+        "issuer": {
+          "description": "Certificate issuer.",
+          "type": "string"
+        }
+      },
+      "x-ms-discriminator-value": "AccessControlService"
+    },
+    "ResourceCertificateDetails": {
+      "description": "Certificate details representing the Vault credentials.",
+      "required": [
+        "authType"
+      ],
+      "type": "object",
+      "properties": {
+        "authType": {
+          "description": "This property will be used as the discriminator for deciding the specific types in the polymorhpic chain of types.",
+          "type": "string"
+        }
+      },
+      "discriminator": "authType"
+    },
+    "ResourceCertificateDetailsResource": {
+      "description": "Certificate details representing the Vault credentials.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ResourceCertificateDetails"

```

Comment:
hmmm, a property named "properties" that returns something of type "...Details". @amarzavery Is this convention regarding extending `Resource` types? Otherwise this mismatch feels a little counter-intuitive.